src/esp/assets/PTexMeshData.cpp

```diff
@@ -512,7 +538,7 @@ void PTexMeshData::parsePLY(const std::string& filename,
       colorOffsetBytes = offsetSoFarBytes;
       offsetSoFarBytes += colorBytes;
     } else {
-      ASSERT(false);
+      CORRADE_ASSERT(false, "Error: unknown.", );
```

Comment:
`CORRADE_ASSERT_UNREACHABLE()` would be more descriptive in this case I think.

Reply:
Good to know. will change.